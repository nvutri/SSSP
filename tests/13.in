c random graph n=32 m=128 lo=0 hi=67108864 seed=1911918788
p sp 32 128
a 32 24 33826744
a 24 6 25002468
a 6 8 8327739
a 8 18 6213706
a 18 27 32302826
a 27 13 15253498
a 13 12 50265916
a 12 3 65060480
a 3 1 40909358
a 1 19 57306788
a 19 5 28607349
a 5 10 6263270
a 10 16 8886059
a 16 14 46181738
a 14 17 16416599
a 17 15 24202688
a 15 25 56830378
a 25 20 50162089
a 20 9 19278771
a 9 30 6938286
a 30 28 44788895
a 28 26 25593453
a 26 4 14500505
a 4 29 8407559
a 29 22 33995155
a 22 2 34168648
a 2 23 20256340
a 23 31 17618997
a 31 11 15959043
a 11 21 16328732
a 21 7 10612042
a 7 32 11925234
a 4 27 61386494
a 11 7 63520083
a 9 16 57135803
a 14 3 48587732
a 23 1 53057321
a 22 17 45767257
a 8 27 8080448
a 12 13 29487987
a 15 17 17577633
a 17 23 38882751
a 18 3 60412207
a 18 19 48066862
a 13 15 11642462
a 5 31 52171133
a 10 30 43099454
a 27 4 27803170
a 30 13 6348530
a 14 7 38789124
a 29 31 13240141
a 25 5 33701876
a 25 4 25001621
a 15 5 13697176
a 1 28 22925989
a 15 14 31051296
a 32 21 10392836
a 15 27 25302798
a 27 17 26225314
a 26 4 66410027
a 19 4 53397221
a 1 19 59184508
a 6 27 24470618
a 20 30 53820970
a 8 15 10009612
a 2 11 9030479
a 20 17 38569048
a 17 12 24129642
a 1 19 15741488
a 3 31 55450819
a 14 5 34187112
a 20 8 28952314
a 12 9 22000448
a 29 10 22623551
a 21 4 9235358
a 24 5 52770365
a 16 26 30272994
a 13 6 13489957
a 3 27 54381912
a 20 19 55496448
a 3 4 21484212
a 14 25 34951940
a 26 29 37180524
a 4 2 24972291
a 16 31 8627288
a 26 9 17973524
a 3 8 47096505
a 12 6 43954669
a 19 11 41832333
a 4 12 23651362
a 1 27 17826812
a 26 16 5700091
a 24 29 23853626
a 29 20 47390239
a 15 7 31904877
a 7 10 39062720
a 20 6 11349028
a 1 23 13340358
a 30 12 45116102
a 4 15 36162675
a 20 31 29604582
a 10 24 18226229
a 24 28 41546105
a 23 8 28708226
a 29 6 12589191
a 19 4 28783897
a 8 16 45038626
a 32 27 47900584
a 9 12 7340287
a 2 8 24655660
a 28 22 57414925
a 13 31 16685104
a 17 8 23224517
a 3 16 10597933
a 24 27 29394054
a 21 8 7216503
a 29 2 65407559
a 3 27 27351969
a 8 17 189094
a 32 12 56675651
a 14 3 65934159
a 24 15 66509945
a 13 15 52667881
a 28 30 39962680
a 31 24 32964519
a 13 9 44950928
a 11 19 44355718
a 22 7 38995552
