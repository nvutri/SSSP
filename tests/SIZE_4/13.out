1 0
2 6.94977e+07
3 1.08674e+08
4 4.45254e+07
5 4.43488e+07
6 4.65703e+07
7 7.38592e+07
8 4.20486e+07
9 6.6493e+07
10 5.06121e+07
11 4.69184e+07
12 6.63673e+07
13 3.30803e+07
14 7.57741e+07
15 4.47228e+07
16 5.42195e+07
17 4.22377e+07
18 4.82623e+07
19 1.57415e+07
20 1.00323e+08
21 6.32471e+07
22 8.03409e+07
23 1.33404e+07
24 6.39239e+07
25 1.01553e+08
26 4.85194e+07
27 1.78268e+07
28 2.2926e+07
29 5.29329e+07
30 6.28887e+07
31 3.09594e+07
32 8.57844e+07
